### PR TITLE
updated doc for squadcast integration

### PR DESCRIPTION
**Pull request by punit-squadcast**
Hi There! 
I am Product solution engineer at Squadcast. We feel that doc needs to be updated for Squadcast integration. Changes has been made, Please review and merge it. 
Thanks & Regards,
Punit

### What does this PR do?

We feel that doc needs to be updated for Squadcast integration

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

attaching screenshot for second image ([2]) available in doc
datadog-webhook

### CODE REVIEW

**Comment by hithwen** [comment]
Hallo, this PR is a draft, is that intended?

File: squadcast/README.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
 ## Overview
 
-Use the Datadog-Squadcast integration to send Datadog alerts to Squadcast and seamlessly take actions on these alerts within the Squadcast platform.
+Use the Datadog-Squadcast integration to send Datadog alerts/incidents to Squadcast and seamlessly take actions on them within Squadcast.
```
Comment (ruthnaebeck) [inline squadcast/README.md:3]:
```suggestion
Use the Datadog-Squadcast integration to send Datadog alerts or incidents to Squadcast and seamlessly take actions on them within Squadcast.
```

File: squadcast/README.md [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
 
 Connect Squadcast to Datadog in order to:
-- Trigger and resolve incidents from Datadog
-- Tackle incidents and set up escalation policies as they occur
-- Set up a daily reminder of who is on-call
+- Trigger, route and resolve alerts/incidents from Datadog
```
Comment (ruthnaebeck) [inline squadcast/README.md:6]:
```suggestion
- Trigger, route, and resolve alerts or incidents from Datadog
```

File: squadcast/README.md [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
+- Tackle alerts/incidents and set up Escalation Policies as they occur
```
Comment (ruthnaebeck) [inline squadcast/README.md:7]:
```suggestion
- Tackle alerts or incidents and set up escalation policies as they occur
```

File: squadcast/README.md [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
+- Define on-call Schedules and set up customisable reminders of who is on-call
```
Comment (ruthnaebeck) [inline squadcast/README.md:8]:
```suggestion
- Define on-call schedules and set up customizable reminders of who is on-call
```

File: squadcast/README.md [hunk 5]
```diff
@@ -1,26 +1,27 @@
...
 
 ## Setup
 
-**Note**: Only the users with Account Owner or Admin privileges can configure services on Squadcast.
-At least one Escalation Policy must be configured before you can add a service.
+**Note**: Only Squadcast users with the right Team-level privileges can configure Services in Squadcast. At least one Escalation Policy must be configured before a Service can be added.
```
Comment (ruthnaebeck) [inline squadcast/README.md:12]:
```suggestion
**Note**: Only Squadcast users with the correct team-level privileges can configure services in Squadcast. At least one escalation policy must be configured before a service can be added.
```

File: squadcast/README.md [hunk 6]
```diff
@@ -1,26 +1,27 @@
...
 
 ### Squadcast
 
 Follow these steps in Squadcast:
 
-1. Open the **Services** page from the sidebar.
+1. Choose the **Team** from the Team-picker on the top.
```
Comment (ruthnaebeck) [inline squadcast/README.md:18]:
```suggestion
1. Choose the **Team** from the team-picker on the top.
```

File: squadcast/README.md [hunk 7]
```diff
@@ -1,26 +1,27 @@
...
 
-2. Choose an existing service, or create a new service by clicking on **Add Service**.
+2. Open the **Services** page from the primary navigation bar on the left.
 
-3. Click on **Alert Sources** and select **Datadog** from the drop down.
+3. Choose an existing Service, or create a new Service by clicking on Add Service.
```
Comment (ruthnaebeck) [inline squadcast/README.md:22]:
```suggestion
3. Choose an existing service, or create a new service by clicking on **Add Service**.
```

File: squadcast/README.md [hunk 8]
```diff
@@ -32,31 +33,60 @@ Follow these steps in Datadog:
...
     }
-    ```
+```
 
-8. Click on "Install Integration" to complete the service integration.
+6. Click on **Save** to complete the service integration.
 
     View the [official documentation][3] from Squadcast for more details on setup.
 
+**Note**: Once the Webhook for Squadcast has been configured, ensure that the same is also selected as a channel within Notify your team in the Monitor’s configuration.
```
Comment (ruthnaebeck) [inline squadcast/README.md:88]:
```suggestion
**Note**: Once the Webhook for Squadcast has been configured, ensure that the same is also selected as a channel under "Notify your team" in the Monitor’s configuration.
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]
Docs review

**Comment by punit-squadcast** [comment]
> Hallo, this PR is a draft, is that intended?

yes!

File: squadcast/README.md [hunk 9]
```diff
@@ -32,31 +33,60 @@ Follow these steps in Datadog:
...
-3. Once the **Webhooks** tile appears, hover and click on "Install".
+3. Once the **Webhooks** tile appears, hover and click on **Install**.
 
 4. Navigate to the **Configuration** tab and scroll to the bottom of the page.
 
-5. Under the section **Name and URL**, enter a meaningful name and paste the **Datadog Webhook URL** provided by Squadcast.
+5. (a) Give the Webhook a name in the **Name** field.
 
-    ![Squadcast Webhook][2]
+   (b) Paste the **Datadog Webhook URL** provided by Squadcast in the URL field.
```
Comment (yzhan289) [inline squadcast/README.md:42]:
```suggestion
   (b) Paste the **Datadog Webhook URL** provided by Squadcast in the **URL** field.
```
to be consistent with `Name` field in bold.

**Review by yzhan289 — COMMENTED** [review]
LGTM, just 1 suggestion

**Review by yzhan289 — APPROVED** [review]

**Review by ruthnaebeck — APPROVED** [review]
👍  for docs

**Comment by punit-squadcast** [comment]
Hey! Only Screenshot is updated, content are not in https://docs.datadoghq.com/integrations/squadcast/ , could you please have a look to this?

**Comment by yzhan289** [comment]
@punit-squadcast, thanks for letting us know! Looks like it is updated now. 👍

**Pull request merged**